src/olympia/translations/forms.py

```diff
@@ -25,56 +27,67 @@ class TranslationFormMixin(object):
     default locale.
     """
 
-    def __init__(self, *args, **kw):
-        super(TranslationFormMixin, self).__init__(*args, **kw)
-        self.error_class = self.error_class_
-        self.set_default_locale()
+    def __init__(self, *args, **kwargs):
+        kwargs['error_class'] = LocaleErrorList
+        super(TranslationFormMixin, self).__init__(*args, **kwargs)
+        self.set_locale_field_defaults()
 
-    def set_default_locale(self):
+    def set_locale_field_defaults(self):
         locale = to_language(default_locale(self.instance))
-        for field in self.fields.values():
-            field.default_locale = locale
-            field.widget.default_locale = locale
+        for field_name, field in self.fields.items():
+            if isinstance(field, _TransField):
+                field.set_default_values(
+                    field_name=field_name,
+                    parent_form=self,
+                    default_locale=locale,
```

Comment:
Is this you slowly rewriting the code in [Black](https://github.com/mozilla/addons-server/issues/8617) style? 🤣

Reply:
:rofl: no quiet… I fixed it in eab651223c as well.